ui/selectors/multichain.ts

```diff
@@ -370,12 +375,15 @@ export const getMultichainCoinRates = (state: MultichainState) => {
   return state.metamask.rates;
 };
 
-function getBtcCachedBalance(state: MultichainState) {
+function getNonEvmCachedBalance(state: MultichainState) {
   const balances = getMultichainBalances(state);
   const account = getSelectedInternalAccount(state);
-  const asset = getMultichainIsMainnet(state)
-    ? MultichainNativeAssets.BITCOIN
-    : MultichainNativeAssets.BITCOIN_TESTNET;
+  const network = getMultichainCurrentNetwork(state);
+
+  const asset =
```

Comment:
Just to be extra clear:

```suggestion
  // We assume that there's at least one asset type in and that is the native
  // token for that network.
  const asset =
```